src/xeus_calc_interpreter.cpp

```diff
@@ -189,7 +197,7 @@ namespace xeus_calc
         {
             nl::json jresult;
             publish_stream("stderr", err.what());
-            publish_execution_error("wrong character", "", {});
+            //publish_execution_error(err.what(), "", {});
```

Comment:
What is the reason for using `publish_stream` instead of `publish_execution_error`? Also, if you comment it out, better remove it. It is still on the git history if you want it back.